changelog/8347.bugfix.rst

```diff
@@ -0,0 +1 @@
+Fix ``GONGMagneogramMap`` for headers which have standard ``DATE-OBS`` keys vs ones which have split ``DATE-OBS`` & ``TIME-OBS``.
```

Comment:
```suggestion
Fix ``GONGMagnetogramMap`` for headers which have standard ``DATE-OBS`` keys vs ones which have split ``DATE-OBS`` & ``TIME-OBS``.
```
I'd want this to be a link, but even if it's not, it should at least be spelled correctly =)